app/scripts/metamask-controller.js

```diff
@@ -22,7 +22,8 @@ module.exports = class MetamaskController {
     this.idStore.setStore(this.ethStore)
     this.messageManager = messageManager
     this.publicConfigStore = this.initPublicConfigStore()
-    this.configManager.setCurrentFiat('USD')
+    var currentFiat = this.configManager.getCurrentFiat() ? this.configManager.getCurrentFiat() : 'USD'
```

Comment:
Lines like this can be written more tersely without teranry operators, since any falsy value would be default to USD:
```
var currentFiat = this.configManager.getCurrentFiat() || 'USD'
```